rules-tests/CodeQuality/Rector/ClassMethod/InlineArrayReturnAssignRector/Fixture/include_empty_assign.php.inc

```diff
@@ -2,14 +2,13 @@
 
 namespace Rector\Tests\CodeQuality\Rector\ClassMethod\InlineArrayReturnAssignRector\Fixture;
 
-final class KeepComments
+final class IncludeEmptyAssign
 {
-    public function getPerson()
+    public function run($person2)
```

Comment:
when param named `$person` sshould be skipped

Reply:
Indeed :+1:  Could you handle it? I'm off soon

Reply:
I will check if any regression if conflict with existing params

Reply:
see PR:

- https://github.com/rectorphp/rector-src/pull/7273